CougSat1-Ground/src/space/cougs/ground/packetprocessing/downlinkpackets/Telemetry.java

```diff
@@ -0,0 +1,102 @@
+package space.cougs.ground.packetprocessing.downlinkpackets;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+
+import space.cougs.ground.satellites.CougSat;
+import space.cougs.ground.utils.FileUtils;
+import space.cougs.ground.utils.Units;
+
+public class Telemetry extends DownlinkPacket {
+
+	public static final int ID = 0x08;
+	private FileInputStream inFile = null;
```

Comment:
Not necessary to store the FileInputStream as a member variable, move the method